### PR TITLE
Add recipies for aliveness service

### PR DESCRIPTION
**Pull request by julianschuler**
This PR adds recipies for building and starting the aliveness service.

Do not merge yet, as the branch and commit hash used for building the aliveness executable has to be adjusted after HULKs/hulk#90 is merged.

### CODE REVIEW

File: recipes-hulks/aliveness/aliveness_git.bb [hunk 1]
```diff
@@ -0,0 +1,190 @@
+SUMMARY = "aliveness"
+HOMEPAGE = "https://hulks.de"
+LICENSE = "GPL-3.0-only"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/git/COPYING;md5=5b4473596678d62d9d83096273422c8c"
+
+inherit cargo pkgconfig
```
Comment (h3ndrk) [inline recipes-hulks/aliveness/aliveness_git.bb]: Remove `pkgconfig`?

File: recipes-hulks/aliveness/aliveness_git.bb [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    crate://crates.io/windows_x86_64_gnullvm/0.42.0 \
+    crate://crates.io/windows_x86_64_msvc/0.42.0 \
+    crate://crates.io/zbus/3.7.0 \
+    crate://crates.io/zbus_macros/3.7.0 \
+    crate://crates.io/zbus_names/2.5.0 \
+    crate://crates.io/zvariant/3.10.0 \
+    crate://crates.io/zvariant_derive/3.10.0 \
+"
+
+do_install:append () {
```
Comment (h3ndrk) [inline recipes-hulks/aliveness/aliveness_git.bb]: Move above `SRC_URI` (containing the crates) so it does not get overlooked?

**Review by h3ndrk — CHANGES_REQUESTED** [review]

File: recipes-hulks/aliveness/aliveness_git.bb (hunk 1 above)
Comment (schmidma) [reply recipes-hulks/aliveness/aliveness_git.bb]: I do not know why, but `hula` needs that. Thus, this was a copy paste... I removed it

File: recipes-hulks/aliveness/aliveness_git.bb (hunk 2 above)
Comment (schmidma) [reply recipes-hulks/aliveness/aliveness_git.bb]: ...ok

**Review by h3ndrk — APPROVED** [review]

**Pull request merged**